doc/UserGuide.md

```diff
@@ -53,18 +53,18 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
-### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
+### Finding all persons containing any keyword in their name or tags: `find`
+Finds persons whose names or tags contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
+> The search is case sensitive, the order of the keywords does not matter, only the name and tags are searched, 
 and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
 * `find John`<br>
   Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find Betsy Tim John friend`<br>
+  Returns Any person having names or tags `Betsy`, `Tim`, `John` or `friend`
```

Comment:
This could be made clearer by stating "tags" after "or" (i.e. Returns Any person having names 'Betsy', 'Tim', 'John' or tag 'friend')

Reply:
Technically, if a tag is named 'Betsy', it will be returned as well.